__tests__/jest/jsonUtils/models.js

```diff
@@ -11,6 +11,12 @@ describe('generateID', () => {
   it('is unique', () => {
     expect(generateID()).not.toBe(generateID());
   });
+  it('seed generates different ID', () => {
+    expect(generateID('test')).not.toBe(generateID('test'));
+  });
+  it('hardcoded seed generates same ID', () => {
+    expect(generateID('test', true)).toBe(generateID('test', true));
+  });
```

Comment:
👍 thanks for the tests

Reply:
🙂  Just a thought, could it be worth having React tree to Sketch JSON test cases in the repo? Probably wouldn't work in a CI though.

Reply:
I'd say it's more interesting to have E2E tests: actually rendering the components and checking it's what we expect using the Sketch API (there are some already: https://github.com/airbnb/react-sketchapp/tree/master/__tests__/skpm)

Reply:
Ok, will leave it out for this PR I think, but will look into building some more E2E test cases soon. Would be really nice to have E2E tests + Sketch builds working in a Linux CI in the future.